cg/models/deliverables/metric_deliverables.py

```diff
@@ -119,6 +122,29 @@ def set_sample_ids(cls, _, values: dict) -> set:
         return set(sample_ids)
 
 
+class ConditionMetricsDeliverables(BaseModel):
+    """Specification for a metric deliverables file with conditions sets."""
+
+    metrics_: List[MetricsBase] = Field(..., alias="metrics")
+
+    @validator("metrics_", always=True)
+    def qc_pass(cls, metrics: List[MetricsBase]) -> str:
+        """Verify that metrics met QC conditions."""
+        failed_metrics: List = []
+        for metric in metrics:
+            if metric.condition is not None:
+                try:
+                    qc_function: Callable = getattr(operator, metric.condition.norm)
+                except AttributeError as error:
+                    raise CgError(
+                        f"{metric.condition.norm} is not an accepted operator for QC metric conditions."
```

Comment:
This should be validated in `MetricCondition`.

Reply:
good idea!